packages/gguf/src/cli.ts

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env node
+
+import { GGMLQuantizationType, gguf } from ".";
+
+interface PrintColumnHeader {
+	name: string;
+	maxWidth?: number;
+	alignRight?: boolean;
+}
+
+const mapDtypeToName = Object.fromEntries(Object.entries(GGMLQuantizationType).map(([name, value]) => [value, name]));
+
+async function main() {
+	const ggufPath = process.argv[2];
+	const { metadata, tensorInfos } = await gguf(ggufPath, {
+		allowLocalFile: true,
+	});
+
+	// TODO: print info about endianess
```

Comment:
do we still need this todo ?

Reply:
Yes, would be nice if we can make it outputs the same as `gguf-py` script